html/dom/elements/global-attributes/the-title-attribute-lf.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>the title attribute - allows "LF"</title>
+<link rel="help" href="http://w3c.github.io/html/dom.html#the-title-attribute">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<body>
+  <h1>the title attribute allows "LF" characters</h1>
+  
```

Comment:
You need to remove the trailing whitespace (here and two other places), or the auto-checking thing won't allow this to be merged.